.env.example

```diff
@@ -0,0 +1,12 @@
+MAINNET_RPC_URL=https://example.com
+TARGET_RPC_URL=https://example.com
+PRIVATE_KEY=0x2......
+TARGET_CHAIN=23
+TARGET_ADDRESS=0xbd335c16c94be8c4dd073ae376ddf78bec1858df
+PRICE_FEED=0xba74737a078c05500dd98c970909e4a3b90c35c6
+PRICE_FEED_SENDER=0xf7d4e7273e5015c96728a6b02f31c505ee184603
+
+EXECUTION_ENDPOINT=https://execution
+VAULT_USER_LTV_TRACKER_CONTRACT_ADDRESS=0x...
+GRAPH_API_URL=https://graph
+VAULT=0x...
```

Comment:
Please clean up the variables here. We have MAINNET_RPC_URL and EXECUTION_ENDPOINT that are the same. Also separate by blocks the configuration for different periodic tasks